articles/azure-app-configuration/push-kv-devops-pipeline.md

```diff
@@ -101,3 +102,7 @@ Create multiple instances of the Azure App Configuration Push task within the sa
 **Why am I receiving a 409 error when attempting to push key-values to my configuration store?**
 
 A 409 Conflict error message will occur if the task tries to remove or overwrite a key-value that is locked in the App Configuration store.
+
+**Why am I getting the "crypto.default.randomFillSync is not a function" error when attempting to push key values?**
```

Comment:
Can we move this and the 409 error one under the troubleshooting section? They are not really FAQ.

Suggested change:
Why am I getting the "_crypto.default.randomFillSync is not a function" error when running the task?